modules/Utils.js

```diff
@@ -179,7 +179,7 @@ class Utils {
         for(const entry of query.entries) {
             let url;
             if (entry.url == null) url = entry.webpage_url;
-            else url = (entry.ie_key != null && entry.ie_key === "Youtube") ? "https://youtube.com/watch?v=" + entry.url : entry.url;
+            else url = entry.url;
```

Comment:
(this also breaks a test)

Reply:
This appears to be a mistake by the developer. If you log the input to the ternary statement you will see that it doesn't work as intended. The url is duplicated for the reproducible test case that I provided.. which means it is effectively always false.

Again, this statement appeared to have no effect so if you can link the failing test I will double check on it.